tools/SourceKit/include/SourceKit/Core/LangSupport.h

```diff
@@ -648,6 +648,17 @@ struct RelatedIdentsInfo {
   ArrayRef<std::pair<unsigned, unsigned>> Ranges;
 };
 
+/// Represent one branch of an if config.
+/// Either `#if`, `#else` or `#elseif`.
+struct IfConfigInfo {
+  unsigned Offset;
+  bool IsActive = false;
```

Comment:
Why do you specify an explicit default value for `IsActive` but not for `Offset`. IIRC both are initialized with unknown values if you don’t set a default value and in both cases you always override the default value.